README.md

```diff
@@ -42,7 +42,7 @@ Iris doesn't support Forge. This is for a few reasons:
 
 ## What's the current state of development?
 
-Iris has a public stable release for 1.16 that works with a custom version of Sodium, and a public beta release for 1.17 that currently does not work with Sodium (currently, there is no official version of Sodium for 1.17). Iris is still in heavy development and gets new improvements every week, and is progressing very rapidly!
+Iris has a public stable releases for 1.16 and 1.17 that work with a custom version of Sodium. Iris is still in heavy development and gets new improvements every week, and is progressing very rapidly!
```

Comment:
It says "that work with"
Could you update it to "that works with"

Reply:
Iris has public stable releases for 1.16 and 1.17 that work with custom versions of Sodium. Iris is still in heavy development and gets new improvements every week, and is progressing very rapidly!

This would be grammatically correct